src/ui/SWMM/frmSummaryReport.py

```diff
@@ -227,9 +236,14 @@ def cboType_currentIndexChanged(self, newIndex):
                                 if header_count == total_num_headers:
                                     if temp_header_list[1] == 'Node':
                                         temp_header_list.remove('Node')
-                                    if temp_header_list[i+1] == '10^6' and temp_header_list[i+2] == 'gal':
-                                        temp_header_list[i+1] = '10^6 gal'
+                                    if temp_header_list[i+1] == '10^6':
+                                        if temp_header_list[i+2] == 'gal':
+                                            temp_header_list[i+1] = '10^6 gal'
+                                        elif temp_header_list[i+2] == 'ltr':
+                                            temp_header_list[i+1] = '10^6 ltr'
```

Comment:
It would be good to think about a more comprehensive approach for handling units and unit conversions.

Reply:
Yeah, I agree.

Reply:
I altered the check slightly but recognize that it's still not a comprehensive approach.